src/java/org/apache/cassandra/db/partitions/PartitionUpdate.java

```diff
@@ -93,13 +93,24 @@ public interface PartitionUpdate extends Partition
      */
     int operationCount();
 
+    // FIXME review
+    /**
+     * The accumulated BTree size of the data contained in this update.
+     *
+     * @return the accumulated BTree size of the data contained in this update.
+     */
+    long accumulatedDataSize();
+
```

Comment:
For Review:
`accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. The implementation has moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.